src/components/http-request/http-request.test.ts

```diff
@@ -352,4 +355,91 @@ describe('probingHTTP', () => {
       expect(res.status).to.eq(200)
     })
   })
+
+  describe('Unit test', () => {
+    describe('generateRequestChainingBody', () => {
+      it('should generate request chaining body', () => {
+        // arrange
+        type TestTable = {
+          body: Record<string, any>
+          responses: ProbeRequestResponse[]
+          expected: Record<string, any>
+        }
+        const testTables: TestTable[] = [
+          {
+            body: {
```

Comment:
Can you please also add a test for when the body is just string, not an object.

Reply:
Ok. Working on it.

Reply:
Added here [60c1bb7](https://github.com/hyperjumptech/monika/pull/929/commits/a9696b888163ef69bf635e409dce067c85af631d).